docs/source/examples/plot_sweep1d.py

```diff
@@ -0,0 +1,20 @@
+"Demonstrates manual and auto sweeping with plot_sweep1d"
+import numpy as np
+from gpkit import Model, Variable, units
+from gpkit.interactive import plot_sweep1d
+
+x = Variable("x", "m", "Swept Variable")
+y = Variable("y", "m^2", "Cost")
+m = Model(y, [y >= (x/2)**-0.5 * units.m**2.5 + 1*units.m**2, y >= (x/2)**2])
+
+# manual sweep
+f, ax = plot_sweep1d(m, x, np.linspace(1, 3, 20), y)
+ax.set_title("Manually swept, 20 points")
+f.show()
+f.savefig("plot_sweep1d.png")
+
+# autosweep
```

Comment:
looking at this example it's awfully hard to tell what causes a sweep to be manual or auto. Without yet knowing, all I can infer is that it's controlled via the sweep argument being either a linspace or a 3-tuple. (what about the ambiguous case of a 3-element manual sweep?)

I think it would pay to be more explicit here, and maybe even separate the sweeping from the plotting, so that `plot_sweep1d` plots a model that has already been swept, or takes in a solution array, or is made to be a method of solution array.